### PR TITLE
Pull Request for New element Epicor by IanGSalmon

### PR DESCRIPTION
**Pull request by IanGSalmon**
Export of Private element Epicor to CE Community

### CODE REVIEW

**Comment by kiawnna** [comment]
@IanGSalmon I'm going to give this one to Ricky to review, as he has more Ground to Cloud experience. I'll check out the other two. :) Thanks!
CC: @RickyAmparoCloud

File: epicor/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---------------------------
+Author Name:IanGSalmon
+**Epicor10 Element**  
+This element allows the creation of an element instance for the Epicor10, on-prem API. In order to implement this element, you will need a method for exposing the local network with access to the Epicor10 API, such as Ground2Cloud.  
+#
```
Comment (kiawnna) [inline epicor/README.md:5]: @IanGSalmon Can you provide a link to the vendor's docs in here? Also, maybe link to our Ground to Cloud docs?

**Review by kiawnna — CHANGES_REQUESTED** [review]
@IanGSalmon GREAT! I have one small comment, then I'll merge it to the community!

File: epicor/README.md (hunk 1 above)
Comment (IanGSalmon) [reply epicor/README.md:5]: Added

**Pull request merged**